server/views/pages/bookedtoday/arrivals/searchforexisting/singleExistingRecordFound.njk

```diff
@@ -40,11 +24,12 @@
                 }) }}
 
                 <p>
-                    <a class="govuk-body govuk-link govuk-link--no-visited-state" href="/confirm-arrival/choose-prisoner" data-qa="choose-prisoner">
+                    <a class="govuk-body govuk-link govuk-link--no-visited-state"
+                       href="/confirm-arrival/choose-prisoner" data-qa="choose-prisoner">
                         Return to list of prisoners arriving today
                     </a>
-                </p> 
-            </div>   
+                </p>
+            </div>
         </main>
```

Comment:
worth checking the html validator when modifying pages - this won't validate due to multiple main elements